addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -822,6 +847,15 @@ export class Rtc extends Record {
         if (camera) {
             await this.toggleVideo("camera");
         }
+        this.cleanups.push(
+            // only register the beforeunload event if there is a call as FireFox will not place
+            // the pages with beforeunload listeners in the bfcache.
+            subscribe(browser, "beforeunload", (event) => {
+                if (this.store.env.services["multi_tab"].isOnLastTab) {
+                    event.preventDefault();
+                }
+            })
+        );
```

Comment:
When the call is recovered by another tab, the `isOnLastTab` seems to have correct value after dozens of seconds later (~30 seconds I think).

This is very minor problem, totally fine to lack this corner-case.